doc/classes/LinkButton.xml

```diff
@@ -9,11 +9,27 @@
 	<tutorials>
 	</tutorials>
 	<methods>
+		<method name="launch_url">
+			<description>
+				Launches the URL assigned by [member url].
+			</description>
+		</method>
 	</methods>
 	<members>
 		<member name="text" type="String" setter="set_text" getter="get_text" default="&quot;&quot;">
+			Sets the label to the specified text.
 		</member>
 		<member name="underline" type="int" setter="set_underline_mode" getter="get_underline_mode" enum="LinkButton.UnderlineMode" default="0">
+			Sets the underline mode. See [LinkButton.UnderlineMode] for the available modes.
+		</member>
+		<member name="url" type="String" setter="set_url" getter="get_url" default="">
+			Sets a URL for this [LinkButton]. If a URL has been set, it will be launched automatically with the default program of the operating system on click.
+			Examples
+			[codeblock]
+			set_url("www.godotengine.com")
```

Comment:
Does it work if you omit the `http://` prefix? (I haven't tested, just making sure there wasn't an omission here.)

Reply:
Yeah, works as expected with `http://` (opens the webpage, only tested in windows).